app/src/main/java/de/tum/in/tumcampusapp/database/TcaDb.java

```diff
@@ -106,7 +106,7 @@
     private static TcaDb instance;
 
     public static synchronized TcaDb getInstance(Context context) {
-        if (instance == null || !instance.isOpen()) {
+        if (instance == null) {
```

Comment:
The fix is not that easy...
The `|| !instance.isOpen()` was there for a reason, however I don't exactly recall why.
@psukys introduced it with #584, do you maybe recall why this was important?

Reply:
For the test cases. After each case, the DB is closed to force a flush of all cursors. This somehow does no longer work.